code/upaya/app/controllers/devise/account_type_controller.rb

```diff
@@ -43,7 +43,7 @@ def unconfirmed_representative?
     def success
       redirect_to dashboard_index_path,
                   notice: t('upaya.notices.account_created',
-                            date: (Time.now + 1.year).strftime('%B %d, %Y'))
+                            date: (Time.current + 1.year).strftime('%B %d, %Y'))
```

Comment:
I know the difference is about timezones. Is there some strategy in place for timezone presentation? We've struggled a lot with that on C2, having to go back and make timezones all explicit. Might be worth thinking about this now in the early stages.